### PR TITLE
Release v0.27.5

### PR DESCRIPTION
**Pull request by alexvanin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,47 @@ Changelog for NeoFS Node
...
+- `--generate-key` flag in CLI control commands (#1103)
+- Various unused code (#1123)
+
+### Upgrading from v0.27.4
+Use `--wallet` key in CLI to provide WIF or binary key file.
+
+Replace `NEOFS_STORAGE_SHARD_N_USE_WRITE_CACHE` with 
+`NEOFS_STORAGE_SHARD_N_WRITECACHE_ENABLED` in Storage node config.
+
+Specify `password:xxx` in config file for NeoFS CLI to avoid password input.
```
Comment (carpawell) [inline CHANGELOG.md:45]: ... when using a wallet.

or something like that?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,47 @@ Changelog for NeoFS Node
...
+- Update TZHash library to v1.5.1
+
+### Removed
+- `--wif` and `--binary-key` keys from CLI (#1083)
+- Extended ACL validator moved to SDK library (#1096)
+- `--generate-key` flag in CLI control commands (#1103)
+- Various unused code (#1123)
+
+### Upgrading from v0.27.4
+Use `--wallet` key in CLI to provide WIF or binary key file.
```
Comment (carpawell) [inline CHANGELOG.md:40]: maybe a few words about exactly _removing_ old `binary-key` and `wif` flags to make it clearer for admins?

File: CHANGELOG.md (hunk 1 above)
Comment (fyrchik) [inline CHANGELOG.md:45]: Not sure, but YAML probably requires a space after a colon, so `password: xxx`

**Review by fyrchik — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alexvanin) [reply CHANGELOG.md:45]: Password cannot be applied to anything besides the wallet so there is no other option :thinking:

**Pull request merged**